locales/en-US.json

```diff
@@ -433,5 +433,10 @@
     "footer_source_code": "Source code",
     "footer_original_source_code": "Original source code",
     "footer_modfied_source_code": "Modified Source code",
-    "adminprefs_modified_source_code_url_label": "URL to modified source code repository"
+    "adminprefs_modified_source_code_url_label": "URL to modified source code repository",
+    "crash_page_you_found_a_bug": "It looks like you found a bug in Invidious!",
+    "crash_page_before_reporting": "Before reporting a bug, make sure that you have:",
+    "crash_page_read_the_faq": "looked at the <a href=\"`x`\">Frenquently Asked Queqtions (FAQ)</a>",
```

Comment:
```suggestion
    "crash_page_read_the_faq": "looked at the <a href=\"`x`\">Frenquently Asked Questions (FAQ)</a>",
```

Reply:
@SamantazFox It's not resolved, have you forgotten a commit?

Reply:
Ah, yeah, I totally missed it! thanks for pinging me ^^